Xamarin.Forms.Platform.Android/Renderers/SwipeViewRenderer.cs

```diff
@@ -1318,7 +1318,8 @@ void ProgrammaticallyOpenSwipeItem(OpenSwipeItem openSwipeItem)
 
 		void UpdateIsOpen(bool isOpen)
 		{
-			((ISwipeViewController)Element).IsOpen = isOpen;
+			if (Element is ISwipeViewController swipeViewController)
+				swipeViewController.IsOpen = isOpen;
```

Comment:
Is this going to lead to wrong behavior upstream?

When testing this I noticed this is called from `DisposeSwipeItems`

So the ISwipeViewController that's associated with this renderers keeps the "IsOpen" value set to true.

Is that ok? Does the dispose need to happen in a different order so that IsOpen correctly propagates to SwipeView?


![image](https://user-images.githubusercontent.com/5375137/83575730-6c55fb00-a4ed-11ea-9ffb-0023ca64e763.png)

Reply:
As @samhouts mentioned, the issue was fixed by other PR. The current changes only add a new tests in Core Gallery.